### ISSUE TITLE
Binary hack.exe only works in Windows 32 bit systems

### ISSUE THREAD

**Issue by RDCH106** [gardaud/hack-exe#2]
Recompile it in 32 bit and 64 bit for the actual systems.

Te alternative is use [DosBox](http://www.dosbox.com/) compatibility layer.

Error snapshot:

![image](https://cloud.githubusercontent.com/assets/1483414/20095071/3d483f2c-a5a4-11e6-95ee-40b73e487231.png)

**Comment by gardaud** [comment]
I think this is because Windows tries to run `hack.exe` as a binary (due to the `.exe` extension), rather than running it from a Python interpreter (unless I am mistaken, Windows does not read the `#!` directive).

If you explicitly launch it with python (`python hack.exe`), I expect you would not encounter this issue.

**Comment by RDCH106** [comment]
Aaaaaaaaaaaaaaaa OK!!
Jejejejejeje....

It is not a binary!!! I have opened and it is a python file. The system assign binary executable icon because of the extension.

**Comment by gardaud** [comment]
Indeed, it is a python file. On most systems, the first line (`#!/usr/bin/python`) should enable the shell to run the file as a python program, but that does not seem to be the case on the standard Windows environment.

If there's a way to make this smoother on Windows, I'll happily take any pull requests.

**Comment by RDCH106** [comment]
I have installed with pip:

```
D:\GitHub>pip install hack-exe
Collecting hack-exe
  Using cached hack-exe-0.2.tar.gz
Installing collected packages: hack-exe
  Running setup.py install for hack-exe ... done
Successfully installed hack-exe-0.2
```
When I execute it:

```
D:\GitHub>python hack-exe google.com
D:\Program Files (x86)\Python35-32\python.exe: can't find '__main__' module in 'hack-exe'
```
